bgpd/bgp_conditional_adv.h

```diff
@@ -33,6 +33,13 @@
 extern "C" {
 #endif
 
+/* Macro to log debug message */
+#define bgp_cond_adv_debug(...)                                                \
+	do {                                                                   \
+		if (BGP_DEBUG(cond_adv, COND_ADV))                             \
+			zlog_debug("[BGP-COND-ADV] " __VA_ARGS__);             \
```

Comment:
Do we really need this prefix `[BGP-COND-ADV]`? We already know the function's name and debug is kinda more for advanced operators, and developers, right?

Reply:
sure will remove '[BGP-COND-ADV]' then